packages/bootstrap/scss/menu/_variables.scss

```diff
@@ -1,49 +1,147 @@
+/// The background of active menu item.
+/// @group menu
+$kendo-menu-item-active-bg: null !default;
+/// The text color of active menu item.
+/// @group menu
+$kendo-menu-item-active-text: if($kendo-enable-color-system, color-mix(in srgb, k-color( on-app-surface ) 100%, transparent), rgba( $kendo-menu-text, 1 )) !default;
+/// The border color of active menu item.
+/// @group menu
+$kendo-menu-item-active-border: null !default;
+/// The background gradient of active menu item.
+/// @group menu
+$kendo-menu-item-active-gradient: null !default;
 
+/// The base shadow of focused menu item.
+/// @group menu
 $kendo-menu-item-focus-shadow: inset 0 0 0 3px if($kendo-enable-color-system, color-mix(in srgb, k-color( on-app-surface ) 15%, transparent), rgba( $kendo-menu-text, .15 )) !default;
 
-$kendo-menu-separator-spacing: k-spacing(1) !default;
+/// The background of selected menu item.
+/// @group menu
+$kendo-menu-item-selected-bg: null !default;
+/// The text color of selected menu item.
+/// @group menu
+$kendo-menu-item-selected-text: null !default;
+/// The border color of selected menu item.
+/// @group menu
+$kendo-menu-item-selected-border: null !default;
+/// The background gradient of selected menu item.
+/// @group menu
+$kendo-menu-item-selected-gradient: null !default;
 
+/// The background of the menu scroll buttons.
+/// @group menu
 $kendo-menu-scroll-button-bg: $kendo-menu-bg !default;
+/// The text color of the menu scroll buttons.
+/// @group menu
 $kendo-menu-scroll-button-text: if($kendo-enable-color-system, color-mix(in srgb, k-color( on-app-surface ) 50%, transparent), rgba( $kendo-menu-text, .5 )) !default;
+/// The border color of the menu scroll buttons.
+/// @group menu
 $kendo-menu-scroll-button-border: $kendo-menu-border !default;
+/// The background gradient of the menu scroll buttons.
+/// @group menu
 $kendo-menu-scroll-button-gradient: null !default;
 
+/// The background of hovered menu scroll buttons.
+/// @group menu
 $kendo-menu-scroll-button-hover-bg: null !default;
+/// The text color of hovered menu scroll buttons.
+/// @group menu
 $kendo-menu-scroll-button-hover-text: if($kendo-enable-color-system, color-mix(in srgb, k-color( on-app-surface ) 70%, transparent), rgba( $kendo-menu-text, .7 )) !default;
+/// The border color of hovered menu scroll buttons.
+/// @group menu
 $kendo-menu-scroll-button-hover-border: null !default;
+/// The background gradient of hovered menu scroll buttons.
+/// @group menu
 $kendo-menu-scroll-button-hover-gradient: null !default;
 
 
+// Menu Separator
+
+/// The inline margin of the horizontal menu separator.
+/// @group menu
+$kendo-menu-separator-margin-inline: k-spacing(1);
+/// The block margin of the horizontal menu separator.
+/// @group menu
+$kendo-menu-separator-margin-block: k-spacing(2);;
+/// The border color of  menu separator.
+/// @group menu
+$kendo-menu-separator-border: var(--kendo-color-border, #{$kendo-component-border}) !default;
```

Comment:
I think this should be `if($kendo-enable-color-system, k-color(border), $kendo-component-border)`